charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -1856,6 +1867,10 @@ def add_op_create_replicated_pool(self, name, replica_count=3, pg_num=None,
                        for pool.
         :type pg_num: int
         :raises: AssertionError if provided data is of invalid type/range
+        :param crush_profile: Name of crush profile to use. If not set the
+                              ceph-mon unit handling the broker request will
+                              set its default value.
+        :type crush_profile: str
```

Comment:
Either `:type crush_profile: Optional[str]` or require `crush_profile` in the function signature?